javascriptv3/example_code/cross-services/wkflw-resilient-service/steps-demo.js

```diff
@@ -0,0 +1,458 @@
+/*
```

Comment:
I would like to see better variable names in here. `lb` is sparse, but ok, but would be better as `loadBalancer`.
`c` is used all over the place but it's hard to tell what it means from any particular usage. Is this the `context` being passed in? That would be a better name.
It's not a _huge_ issue but would aid in readability.

Reply:
Normally I would agree, and this is not a strong "but", _but_ `lb` is only used in very close proximity to things like `loadBalancerLoop` and `findLoadBalancer`. And `context` isn't much more explanatory than `c`, although I'll allow it's less clear. I'll be more verbose on the next one.